.golangci.yaml

```diff
@@ -75,6 +75,20 @@ linters-settings:
       # Controller Runtime
       - pkg: sigs.k8s.io/controller-runtime
         alias: ctrl
+      - pkg: k8s.io/apimachinery/pkg/util/runtime
+        alias: utilruntime
+      - pkg: sigs.k8s.io/cluster-api/api/v1beta1
+        alias: clusterv1
+      - pkg: sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1beta1
+        alias: bootstrapv1
+      - pkg: sigs.k8s.io/controller-runtime/pkg/metrics/server
+        alias: metricsserver        
+      - pkg: sigs.k8s.io/cluster-api/errors
+        alias: capierrors    
+      - pkg: sigs.k8s.io/controller-runtime/pkg/client/fake
+        alias: fakeclient       
+      - pkg: sigs.k8s.io/controller-runtime/pkg/client
```

Comment:
I am a bit hesitant with this one. I would remove it. "c" is just not a great choice for any alias, it is more a workaround :D
Did all the code comply to these aliases?

Reply:
Yes it does , I went through it .
Then I'll remove this alias and push the changes

Reply:
> "c" is just not a great choice

How about using `ctrlc` given we already use `ctrl` for controller runtime.

Reply:
that is an idea. But at this point I wouldn't like to enforce this